docs/csharp/tutorials/inheritance.md

```diff
@@ -183,7 +183,7 @@ In designing our `Publication` class, we need to make several design decisions:
 
   By default, a base class can be instantiated by calling its class constructor. Note that we do not have to explicitly define a class constructor. If one is not present in the base class' source code, the C# compiler automatically provides a default (parameterless) constructor.
 
-  For our example, we'll mark the `Publication` class as [abstract](../language-reference/keywords/abstract.md) so that it cannot be instantiated.
+  For our example, we'll mark the `Publication` class as [abstract](../language-reference/keywords/abstract.md) so that it cannot be instantiated. The `Publication` class does not have any `abstract` methods, but the class itself is `abstract`. An `abstract` class without any `abstract` methods indicates that this class represents an abstract concept that is shared among several concrete classes (like a `Book`, `Journal`).
```

Comment:
This seems a bit abrupt, and although it addresses the issue of an abstract class without abstract methods, it doesn't say what an abstract class with abstract methods is. Since we're discussing design decisions, that seems like a major omission. (It's also a major area of confusion.) 
 
I think that what I would do is move the discussion of abstract methods to the next bulleted item, so that the choice becomes inheriting the base class implementation, optionally overriding the base class implementation, or being required to provide an implementation. Note that this can be done on a method-by-method basis, except that if any members are abstract, the type must also be abstract. The abstract member discussion here can be succinct, with a forward-reference to the next section.